src/scenes/Event/scenes/oracle.js

```diff
@@ -657,7 +674,7 @@ export default class OraclePage extends Component {
     }
 
     // Did not select a result
-    if (!isFinalizePhase && currentOptionIdx === -1) {
+    if (!isFinalizePhase && currentOptionIdx === -1 && !isArchived) {
```

Comment:
just disable this entire function `getActionButtonConfig()` for archived oracles.
```
if (isArchived) {
   return;
}
```